config/locales/client.en.yml

```diff
@@ -132,11 +132,17 @@ en:
         title: Unfollow for %{actor} via ActivityPub
         modal_title: Confirm Unfollow
         confirm: Are you sure you want %{actor} to unfollow %{followed_actor}?
+      actor_reject:
+        label: Reject
+        title: Reject follow of %{actor} via ActivityPub
+        modal_title: Confirm Reject Follow
+        confirm: Are you sure you want %{actor} to reject the follow of %{followed_actor}?
```

Comment:
Should this be something like: "Are you you want to reject actor X from following actor Y?"

Reply:
Sure, we can make that perspectival shift.

Reply:
Done https://github.com/angusmcleod/discourse-activity-pub/commit/f2de51c1ae84151b0eb3c00ecad62dea93df67dd